operator/v1/types_dns.go

```diff
@@ -287,6 +287,23 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+
+	// protocolStrategy specifies the protocol to use for upstream DNS
+	// requests. Valid values for protocolStrategy are "TCP" or empty.
+	// If empty, the default behavior of CoreDNS is used, which is currently
+	// the following: use the protocol of the original client request, but
+	// this is subject to change over time. "TCP" specifies that the platform
+	// should use TCP for all upstream DNS requests, even if the client request
+	// uses UDP. "TCP" is useful for UDP-specific issues such as those created by
+	// non-compliant upstream resolvers, but may consume more bandwidth or
+	// increase DNS response time. Note that protocolStrategy only affects
+	// the protocol of DNS requests that CoreDNS makes to upstream resolvers.
+	// It does not affect the protocol of DNS requests between clients and
+	// CoreDNS.
```

Comment:
```suggestion
	// protocolStrategy specifies the protocol to use for upstream DNS
	// requests.
	// Valid values for protocolStrategy are "TCP" and omitted.
	// When omitted, this means no opinion and the platform is left to choose
	// a reasonable default, which is subject to change over time.
	// The current default is to use the protocol of the original client request.
	// "TCP" specifies that the platform should use TCP for all upstream DNS requests,
	// even if the client request uses UDP. 
	// "TCP" is useful for UDP-specific issues such as those created by
	// non-compliant upstream resolvers, but may consume more bandwidth or
	// increase DNS response time. Note that protocolStrategy only affects
	// the protocol of DNS requests that CoreDNS makes to upstream resolvers.
	// It does not affect the protocol of DNS requests between clients and
	// CoreDNS.
```

Some minor changes to the text to fit current convention for wording:
* We tend to say omitted rather than empty
* We have a common prose for when omitted.
* I removed the bit about the default being the CoreDNS default, not sure how useful that really is
Please apply the same doc to both instances

Reply:
Done.